src/enclave/ortoa.conf

```diff
@@ -2,9 +2,15 @@
 # Licensed under the MIT License.
 
 # Enclave settings:
-Debug=1
-NumHeapPages=2048
+
+# snmalloc requires at least 256 KB per enclave thread.
+# Given 16 enclave threads (NumTCS), this implies
+#    minimum heap size = (256 * 1024 * 8) / 4096 = 512 pages.
+# The heap size (4096 pages) is well above the minimum requirement,
+# and accounts for the large number of allocations performed by
+# each enclave thread in the sample.
```

Comment:
If the goal of this PR is to be merged, we should align the math with the actual `NumTCS` we use